Traceback (most recent call last):
  File "skema/program_analysis/model_coverage_report/model_coverage_report.py", line 76, in generate_data_product
    output = func_timeout(10, data_product_function, args=args, kwargs=kwargs)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/func_timeout/dafunc.py", line 108, in func_timeout
    raise_exception(exception)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/func_timeout/py3_raise.py", line 7, in raise_exception
    raise exception[0] from None
  File "skema/program_analysis/model_coverage_report/model_coverage_report.py", line 108, in generate_cast
    cast = fortran_to_cast(source_path, cast_obj=True)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/fortran2cast.py", line 64, in fortran_to_cast
    out_cast = TS2CAST(path).out_cast
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 82, in __init__
    self.out_cast = self.generate_cast()
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 87, in generate_cast
    modules = self.run(self.root_node)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 101, in run
    modules.append(self.visit(context))
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 127, in visit
    return self.visit_module(node)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 184, in visit_module
    child_cast = self.visit(child)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 139, in visit
    return self.visit_assignment_statement(node)
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/skema/program_analysis/CAST/fortran/ts2cast.py", line 707, in visit_assignment_statement
    left, _, right = node.children
ValueError: too many values to unpack (expected 3)
